recipe/meta.yaml

```diff
@@ -27,6 +27,7 @@ requirements:
     - python
     - aiormq >=6.7.5,<7
     - yarl
+    - exceptiongroup >=1.0.0,<2.0.0
```

Comment:
I think this would need < 3.11 specified since exceptiongroup backports from python 3.11

Reply:
looks like it is needed for all as the code imports it without condition